plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -112,6 +112,11 @@ public BigQueryClient(
         this.configProjectId = requireNonNull(configProjectId, "projectId is null");
     }
 
+    public Optional<RemoteDatabaseObject> toRemoteDataset(DatasetId datasetId)
```

Comment:
protected

Reply:
The other existing `toRemoteDataset` is public, I wanted to be consistent with it. Should I also change it there?

Reply:
@wendigo PTAL, @hashhar can you run test with secrets again?

Reply:
I can only mirror this PR

Reply:
@hashhar ping

Reply:
let's keep it public just for consistency's sake.